GoogleDataTransport.podspec

```diff
@@ -41,7 +41,7 @@ Shared library for iOS SDK data transport needs.
 
   s.dependency 'GoogleUtilities/Environment', '~> 7.7'
   s.dependency 'GoogleUtilities/Logger', '~> 7.7'
-  s.dependency 'nanopb', '~> 2.30908.0'
+  s.dependency 'nanopb', '>= 2.30908.0', '< 2.30910.0'
```

Comment:
Why is the version specifier different from https://github.com/firebase/firebase-ios-sdk/pull/9765/?

> From `FirebaseFirestore.podspec` in https://github.com/firebase/firebase-ios-sdk/pull/9765/:
> ```diff
>   - s.dependency 'nanopb', '~> 2.30908.0'
>   + s.dependency 'nanopb', '~> 2.30909.0'
> ```

Reply:
It looks like GDT is allowing 0.3.9.8 and 0.3.9.9 but Firebase pods are allowing ~> 0.3.9.9.

Reply:
This is intentional so that the latest GDT will remain with Firebase 8

Reply:
Ahh I see. Very cool 👍